.github/workflows/update_protos.yml

```diff
@@ -46,12 +46,14 @@ jobs:
           BOOST_ROOT: ${{ github.workspace }}/boost/boost
       - name: cleanup
         run: rm -rf boost/ && rm -rf vcpkg/
+      - name: Update proto tag
+        run: sed -i -e 's/VIAMCPPSDK_API_PROTO_LABEL "v[0-9]*\.[0-9]*\.[0-9]*"/VIAMCPPSDK_API_PROTO_LABEL "${{ github.event.client_payload.tag }}"/' src/viam/sdk/CMakeLists.txt
       - name: Add + Commit + Open PR
         uses: peter-evans/create-pull-request@v3
         with:
           commit-message: '[WORKFLOW] Updating protos from ${{ github.event.client_payload.repo_name }}, commit: ${{ github.event.client_payload.sha }}'
           branch: 'workflow/update-protos'
-          add-paths: src/viam/api/*
+          add-paths: src/viam/api/*, src/viam/sdk/CMakeLists.txt
```

Comment:
Unfortunately this is only partially untested. I know that the `sed` command works locally and I know that `tag` is the correct payload name based on the `api` job, but we're currently already caught up in proto here. I have fairly high confidence that this will work and I believe the consequences of being wrong are low (next time `update_protos` runs we'd either see it fail or improperly update the `VIAMCPPSDK_API_PROTO_LABEL` variable, either of which should be fairly straightforward to fix), but if anyone has concerns let me know!